src/create.cpp

```diff
@@ -40,27 +42,61 @@ unsigned Create::run()
         throw rest::Error(ret.error());
     }
 
-    std::string assetJson = m_request.body();
-
-    auto ret = AssetManager::createAsset(assetJson, user.login());
-    if (!ret) {
-        auditError(ret.error());
-        throw rest::errors::Internal(ret.error());
+    cxxtools::SerializationInfo si;
+    std::stringstream           jsonIn;
```

Comment:
jsonIn(m_request.body());
And inside try block (otherwise this stream in parent scope...)